packages/react-native-worklets/android/build.gradle

```diff
@@ -72,7 +72,6 @@ def REACT_NATIVE_MINOR_VERSION = getReactNativeMinorVersion()
 def WORKLETS_VERSION = getWorkletsVersion()
 def IS_NEW_ARCHITECTURE_ENABLED = isNewArchitectureEnabled()
 def IS_REANIMATED_EXAMPLE_APP = safeAppExtGet("isReanimatedExampleApp", false)
-def REANIMATED_PROFILING = safeAppExtGet("enableReanimatedProfiling", false)
```

Comment:
Currently, `ReanimatedSystraceSection` is used only in `react-native-reanimated` so we don't need that in `react-native-worklets`. We can change that once we need systraces in worklets.